libs/api/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="boost" version="1.75.0.0" targetFramework="native" />
+  <package id="boost" version="1.80.0" targetFramework="native" />
```

Comment:
external\ebpf-verifier\CMakeLists.txt installs boost 1.78.0.
Shouldn't we try to use the same boost version throughout (and make sure boost is only installed once)?